gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -401,3 +401,49 @@ def add_arguments(self, parser):
               'by chromosome (one table per chromosome) which is located at: '
               'gcp_variant_transforms/data/partition_configs/'
               'homo_sapiens_default.yaml'))
+
+
+class BQtoVcfOptions(VariantTransformsOptions):
+  """Options for BigQuery to VCF pipelines."""
+
+  def add_arguments(self, parser):
+    # type: (argparse.ArgumentParser) -> None
+    parser.add_argument(
+        '--output_file',
+        required=True,
+        help='The full path of the VCF file to store the result.')
+    parser.add_argument(
+        '--input_table',
```

Comment:
Just FYI: longer term we also want to support wildcard tables (for partitioned use case). I don't think anything needs to be changed here though.